examples/time-elapsed-example.py

```diff
@@ -12,9 +12,9 @@
 
 
 # Make sure you are using the same name that you used when uploading the image
-start_time=time.time()
+start_time=time.time() # Using the time that we imported at the start. start_time equals time.
 RPC.update(large_image="LARGE_IMAGE_HERE", large_text="Programming B)",
```

Comment:
I was hoping for a bit of explanation on small and large images but this is fine :)

Reply:
There is already a image example so I felt like I didn't need it 😅